microsoft/botframework-solutions
# [TypeScript][BotBuilder-Skills] Add SkillManifestGenerator

## Description
- Add `SkillManifestGenerator` to botbuilder-skills library.
- Add compatibility with `restify`.
- Add tests for `SkillManifestGenerator`.

## Testing Steps
1. Go to `./lib/typescript` and open a PowerShell console.
1. Run `rush install` to install the depdendencies.
1. Run `rush rebuild` to build the projects.
1. Run `rush test` to run the tests.
1. Check that all tests work fine.

## Checklist
